### PR TITLE
Adds NC Bulker vessel.

### PR DESCRIPTION
**Pull request by CharcoalShipyard**
## About the PR

We have created a small mining ship named NC Bulker.

## Why / Balance

Our shipyard (scrapyard) company always wanted to design a new ship for all hard workers on Frontier Station. 

## Technical details

I was following the mapping guide, therefore I haven't made any changes to the code itself.

## Media

image
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**
- add: NC Bulker

### CODE REVIEW

File: Resources/Prototypes/_NF/Shipyard/bulkermax.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+- type: vessel
+  id: BULKER
+  name: NC Bulker
+  description: Small mining vessel for deep space operations
+  price: 35000 #31000ish after the mapinit appraisal
```
Comment (dvir001) [inline Resources/Prototypes/_NF/Shipyard/bulkermax.yml:5]: Keep in mind ships wont pass new price testing unless they are more then 15% added

File: Resources/Maps/Shuttles/bulkermax.yml [hunk 2]
```diff
@@ -0,0 +1,4880 @@
...
+      parent: 1
+      type: Transform
+- proto: KitchenReagentGrinder
+  entities:
+  - uid: 156
+    components:
+    - pos: -9.5,-2.5
+      parent: 1
+      type: Transform
+- proto: LockerCaptain
```
Comment (dvir001) [inline Resources/Maps/Shuttles/bulkermax.yml:3391]: Please use the pre filled locker, this is harder for us to keep balance on ships by using ones you fill yourself

Comment (CharcoalShipyard) [reply Resources/Maps/Shuttles/bulkermax.yml:3391]: Gotcha.

**Comment by Cheackraze** [comment]
why close? was about to start ship reviews and this look really good

**Comment by CharcoalShipyard** [comment]
> why close? was about to start ship reviews and this look really good

I've created a new branch for some reason while I was trying to fix my ship... 
https://github.com/new-frontiers-14/frontier-station-14/pull/566

Thank you for checking my ship!

**Pull request closed without merging**